File: app/models/physician.rb
class Physician < ApplicationRecord
  validates :first_name, presence: true
  validates :last_name, presence: true
  validates :office_name, presence: true
  validates :specialty, presence: true
  validates :address, presence: true
  validates :city, presence: true
  validates :state, presence: true
  #
  # has_many :appointments
  # has_many :users, through: :appointments
  #
  # has_many :physician_lists
  #

  has_many :appointments
  has_many :physicians_with_appointments, through: :appointments, source: :user

  has_many :physician_lists
  has_many :users, through: :physician_lists
end

File: react/src/components/physician.js
import React, { Component } from 'react';

const Physician = props => {
  return (
    <div className="physician">
      <div className='physician-name'>Dr. {props.first_name} {props.last_name}</div>
      <div className='physician-specialty'>{props.specialty}</div>
      <div className='physician-phone_number'>{props.phone_number}</div>
      <div className='physician-office_name'>{props.office_name}</div>
      <div className='physician-address'>{props.address} {props.city} {props.state}</div>
    </div>
  );
}

export default Physician;
